### PR TITLE
Ignore errors when notifying Hipchat

### LINKED ISSUE #34: Failing to notify hipchat should not cause error?
I've been seeing a lot of timeouts today when notifying Hipchat which causes the deployment to fail. Do we want to make this a soft failure?

### PR DESCRIPTION
**Pull request by jszwedko**
Will cause error output, but the play will continue.

Fixes #34

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@
     mode=0644
   changed_when: False
   when: hipchat_token != "" and hipchat_room != ""
+  ignore_errors: True
```
Comment (rafecolton) [inline tasks/main.yml:20]: Should we maybe standardize on our boolean notation?  Per https://github.com/ansible/ansible/issues/2930

Comment (jszwedko) [inline tasks/main.yml:20]: Interesting, we should do a sweep then.

Comment (jszwedko) [inline tasks/main.yml:20]: Mind merging this first and I'll update all boolean values?

Comment (rafecolton) [inline tasks/main.yml:20]: Sure thing!

**Comment by rafecolton** [comment]
Oops, just noticed that this is missing an `ignore_errors: yes` on the actual hipchat deployment start notification (it's on the html rendering task instead)

**Comment by rafecolton** [comment]
Fixing over here => https://github.com/modcloth/ansible-role-modcloth-app/issues/40

**Pull request merged**